app/medInria/medApplication.cpp

```diff
@@ -162,6 +117,33 @@ void medApplication::open(QString path)
 
 void medApplication::initialize()
 {
```

Comment:
This is the code that was previously in the constructor. The only changes here are the call to ```medLogger::initialize()``` and the first lines (```setApplicationName``` etc.) that have been moved at the top because they have to be executed before the initialisation of the logger (it uses the application name to set the log file path).